ui/src/app/components/webcomponents/lab-environments-info.component.ts

```diff
@@ -65,7 +65,7 @@ export class LabEnvironmentsInfoComponent implements OnInit {
           if(res.content.length){
             agent = res.content[0];
             if(env.browser!=null){
-              env.platform = Agent.getPlatformFromOsType(agent.osType);
+              env.platform = agent.getPlatformFromOsType(agent.osType);
```

Comment:
other modules have agent in class properties but here it's captured in variable, plz make it consistency

Reply:
Done, added agent property to the class